obelix/idefix/trigger/request.go

```diff
@@ -0,0 +1,16 @@
+package trigger
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+func ReadBody(res *http.Response) ([]byte, error) {
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read body: %w", err)
+	}
+
+	return body, nil
+}
```

Comment:
As I already said, this function is useless, it will take the same amount of lines using `ReadBody` than directly `i outil.ReadAll`

Reply:
Yes i see but i prefer to rename in specific function